salt/ocdsdata.sls

```diff
@@ -1,12 +1,28 @@
 {% from 'lib.sls' import createuser %}
 
+# Set up the server
+
+/etc/motd:
+  file.managed:
+    - source: salt://system/ocdsdata_motd
+
+ocdsdata-prerequisites  :
+  pkg.installed:
+    - pkgs:
+      - python-pip
+      - python3-pip
+      - python3-virtualenv
+      - virtualenv
+      - postgresql-10
+      - screen
```

Comment:
I think me and @idlemoor tend to use tmux.

Reply:
Interesting. My suspicion is that it doesn't matter too much if people use one or the other, but there is perhaps increased risk of stomping on each others' work. I use screen out of laziness and muscle memory (which doesn't even work when I don't have my external keyboard plugged in!), but I'd be very open to standardising around tmux. Maybe tmux + some docs in the MOTD?